[IMP] Livechat: URL Regex clarification (#10533)
odoo/documentation: content/applications/websites/livechat.rst

place.

Edit the :guilabel:`Welcome Message` to change the message a visitor sees when they open a new chat
session. This message appears as though it is sent by a live chat operator, and acts as both a
greeting and an invitation to continue the conversation.

Edit the :guilabel:`Chat Input Placeholder` to alter the text that appears in the box where visitors
type their replies. This message prompts the visitor to initiate the chat.

The *Channel Header* is the colored bar at the top of the chat window. The :guilabel:`Channel Header
Color` can be changed following the same steps as the :ref:`Livechat button
<livechat/livechat-button>`.

.. figure:: livechat/chat-window.png
   :align: center

   The Livechat Window with a purple header. The chat input placeholder reads "Ask Something..."

.. _livechat/channel-rules-tab:

Channel rules
-------------

The :guilabel:`Channel Rules` tab on the live chat channel detail form determines when the live chat
window opens on the website, by configuring when a :guilabel:`URL Regex` action is triggered, such

> as a page visit.

Reviewer: I was slightly confused by the sentence, so how about something like:

To configure which website user actions open the live chat window, go to the ...

        field that appears when this option is selected).
      - :guilabel:`Hide`: hides the chat button on the page.

   .. tab:: Chatbot

      To include a :doc:`Chatbot <livechat/chatbots>` on this channel, select it from the drop-down
      menu. If the chatbot should only be active when no operators are active, tick the checkbox
      labeled, :guilabel:`Enabled only if no operator`.

   .. tab:: URL Regex

      The *URL Regex* specifies the web pages where this rule should be applied. In the
      :guilabel:`URL Regex` field, input the relative URL of the page where the chat button should
      appear.

      For example, to apply the rule to the URL, `https://mydatabse.odoo.com/shop`, enter `/shop`
      to the :guilabel:`URL Regex` field.

      To apply the rule to *all* pages on the database, enter `/` in the :guilabel:`URL Regex`
      field.

   .. tab:: Open automatically timer

      This field designates the amount of time (in seconds) a page should be open before the chat
      window opens. This field **only** appears if the :guilabel:`Live Chat Button` for this rule is

>       set to :guilabel:`Open automatically`.

Reviewer 2: Nitpick to add a blank line before tab item :)
```suggestion
      set to :guilabel:`Open automatically`.
      
```

      The *Livechat Button* is the icon that appears in the bottom corner of the website. Select
      from one of the following display options:

      - :guilabel:`Show` displays the chat button on the page.
      - :guilabel:`Show with notification` displays the chat button, as well as a floating text
        bubble next to the button.
      - :guilabel:`Open automatically` displays the button and automatically opens the chat window
        after a specified amount of time (designated in the :guilabel:`Open automatically timer`
        field).
      - :guilabel:`Hide` hides the chat button on the page.

   .. tab:: Chatbot

      To include a :doc:`Chatbot <livechat/chatbots>` on this channel, select it from the drop-down

>       menu. If the chatbot should only be active when no operators are active, check the box labeled

Reviewer 3: ```suggestion
      menu. If the chatbot should only be active when no operators are active, tick the checkbox labeled,
```

      :guilabel:`Enabled only if no operator`.

   .. tab:: URL Regex

      The *URL Regex* specifies the web pages where this rule should be applied. In the
      :guilabel:`URL Regex` field, input the relative URL of the page where the chat button should
      appear.

      For example, to apply the rule to the URL, `https://mydatabse.odoo.com/shop`, enter `/shop`
      to the :guilabel:`URL Regex` field.

      To apply the rule to *all* pages on the database, enter `/` in the :guilabel:`URL Regex`
      field.

   .. tab:: Open automatically timer

      This field designates the amount of time (in seconds) a page should be open before the chat
      window opens. This field **only** appears if the :guilabel:`Live Chat Button` for this rule is
      set to :guilabel:`Open automatically`.
   .. tab:: Country

>       If this channel should only be available to site visitors in specific countries, add them to

Reviewer 3: ```suggestion
      If this channel should **only** be available to site visitors in specific countries, add them to
```

      the :guilabel:`Country` field. If this field is left blank, the channel is available to all
      site visitors, regardless of location.

.. note::
   To track the geographical location of visitors, *GeoIP* **must** be installed on the database.
   While this feature is installed by default on *Odoo Online* databases, *On-Premise* databases
   require additional :doc:`setup steps </administration/on_premise/geo_ip>`.

.. _livechat/widget-tab:

> Widget

Reviewer 3: ```suggestion
Widget tab
```

------

The :guilabel:`Widget` tab on the live chat channel detail form provides the code for a website
widget. This code can be added to a website to provide access to a live chat window.

.. tip::
   The live chat widget can be added to websites created through Odoo by navigating to
   :menuselection:`Website app --> Configuration --> Settings`. Then, scroll to the
   :menuselection:`Email & Marketing` section. In the :guilabel:`Channel` field, select the channel

>    to add to the site. Click :guilabel:`Save` to apply.

Reviewer 3: This 'Email & Marketing' section, nor the 'Channel' field are present in the 16.0 runbot I'm using for the review. The closest field I found was a 'Channel' field, beneath the LiveChat setting, in the 'How to configure the Twitter API access'... But I don't believe this is the same thing. Please double-check the accuracy of this statement during your revisions.

Author: It is the same section, the changed the heading (I believe in 16.2 or 16.3). I have  made the update to reflect the UI

-----------------------

> To join a live chat channel, go to the :menuselection:`Live Chat` app and click the :guilabel:`JOIN`

Reviewer 3: ```suggestion
To join a live chat channel, go to the :menuselection:`Live Chat` app, and click the :guilabel:`JOIN`
```

button on the Kanban card for the appropriate channel.

> Any channel where the user is currently active will show a :guilabel:`LEAVE` button. Click this

Reviewer 3: ```suggestion
Any channel where the user is currently active shows a :guilabel:`LEAVE` button. Click this
```

-----------------------

To join a live chat channel, go to the :menuselection:`Live Chat` app and click the :guilabel:`JOIN`

> button on the Kanban card for the appropriate channel.

Reviewer: totally your preference, but im not sure if it's necessary to mention the Kanban card, as only the kanban cards are displayed on the Livechat main dashboard
```suggestion
button on the desired channel.
```

leaving their current location.

.. tip::
   Conversations can also be accessed by clicking the :guilabel:`Conversations` icon in the menu

>    bar.

Reviewer 3: I think this could use some more clarification, and perhaps the GUI might not be applicable here because it's just an icon -- and the name might not be present. But, if it is, feel free to include. Also, in addition to the clarification, you may want to consider adding an icon to this portion.
